src/customer-service/Factories/LabelFactory.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Data.Sqlite;
+using customer_service.Models;
+using customer_service.Data;
+
+namespace customer_service
+{
+
+    /**
+     * Class: LabelFactory
+     * Purpose: Holds a singleton and can Get labels based on an IncidentTypeId
+     * Author: Matt Kraatz
+     * Methods:
+     *     List<Label> getLabels(int id) - Gets all Labels that match an IncidentType
```

Comment:
`getLabels` should have an uppercase "G" like so: `GetLabels(int id)`

Apologies for being nitpicky :)